blog/managers.py

```diff
@@ -15,9 +15,9 @@ def create_user(self, email, password=None, **extra_fields):
         return self._create_user(email, password, **extra_fields)
 
     def create_superuser(self, email, password, **extra_fields):
-        extra_fields.setdefault('is_superuser', True)
+        extra_fields.setdefault('is_admin', True)
 
```

Comment:
Yesterday we discussed this, you need to use 2 flags, Not only is_admin, but also Is_superuser

Reply:
I deleted by accident